scraper/src/fcc2zim/entrypoint.py

```diff
@@ -153,14 +136,14 @@ def main():
 
     args = parser.parse_args()
 
-    Global.logger.info(f"Starting scraper {VERSION}")
+    Global.logger.info(f"Starting fcc2zim {VERSION}")
 
     set_debug(debug=args.debug)
 
     scraper = Scraper(
-        do_fetch=args.fetch,
-        do_prebuild=args.prebuild,
-        do_build=args.build,
+        do_fetch=strtobool(os.getenv("DO_FETCH", "False")),
```

Comment:
Not a fan of lazy matching for this kind of things. My opinion is that either you know what you're doing or you check the code but this fosters guess and trial.

I don't mind keeping it but if you were just trying to accommodate various developers but have a preference of your own that you always use, you can just test that value.